### PR TITLE
Improve button design

### PR DESCRIPTION
**Pull request by KaspianDev**
image
Pretty basic, matching the original theme pretty well

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -19,3 +19,21 @@ body {
   border-top: 1px solid #007bff;
   border-bottom: 3px solid orangered;
 }
+
+.switch-button {
+  appearance: none;
+  border: none;
+  background: #007bff;
```
Comment (immattdavison) [inline style.css:26]: this matches the document background colour and therefore does not appear like a button.

File: style.css [hunk 2]
```diff
@@ -19,3 +19,21 @@ body {
   border-top: 1px solid #007bff;
   border-bottom: 3px solid orangered;
 }
+
+.switch-button {
```
Comment (immattdavison) [inline style.css:23]: this class should be given a `margin-block`

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (KaspianDev) [reply style.css:23]: What should be the size of the margin?

Comment (immattdavison) [reply style.css:23]: Up to you, just mentioning it as it appears squished into a gap as it is

**Pull request closed without merging**